src/qml/components/CircleProgressBar.qml

```diff
@@ -23,6 +25,42 @@ Item {
         canvas.degree = ratio * 360;
```

Comment:
That was my quick implementation - rotate entire canvas. If you can use angle but don't rotate canvas it will be more logical implementation. Optional fix

Reply:
I think this is a waste of time. And it seems to me that performance might suffer.

Reply:
Ok, skip for now